lib/commands/exam.js

```diff
@@ -254,5 +372,97 @@ module.exports = TestCommand.extend({
       case 'json':
         return fs.readJsonSync(file);
     }
+  },
+
+  /**
+   * Adds additional event handlers to config to enable load balancing.
+   *
+   * @param {object} config
+   * @return {object} events
+   */
+  _addLoadBalancingBrowserSocketEvents(config) {
+    const events = config.custom_browser_socket_events || {};
+
+    events['set-modules-queue'] = function(modules) {
```

Comment:
Should we prefix this event?

Reply:
We wanted to distinguish events which are sent from server side to browser side and from browser side to server side. Events emitted in server side are prefixed with `testem`. If we prefix all events I think it's beneficial as it's all unified but wouldn't it give confusion where we have to distinguish where the events are sent from?

Reply:
Why not prefix on both sides, rather than one side?